src/hardware/canbus/src/node_canbus_translator.cpp

```diff
@@ -183,7 +188,7 @@ class CanBus : public rclcpp::Node {
    public:
     CanBus() : Node("canbus_translator_node") {
         // Can2Ethernet parameters
-        std::string ip = this->declare_parameter<std::string>("ip", "192.168.2.125");
+        std::string ip = this->declare_parameter<std::string>("ip", "192.168.2.126");
```

Comment:
Is this IP documented somewhere? If so, should probably put a comment in specifying where.